### PR TITLE
Add std errors to location mutation

### PR DESCRIPTION
**Pull request by roxy-dao**
#1049 

~~- Currently investigating test failure~~

### CODE REVIEW

File: service/src/location/mod.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use self::{
+pub use self::{
```
Comment (clemens-msupply) [inline service/src/location/mod.rs:1]: Is this needed?

File: graphql/location/src/mutations/update.rs [hunk 2]
```diff
@@ -42,7 +43,9 @@ pub fn update_location(
         Ok(location) => Ok(UpdateLocationResponse::Response(LocationNode::from_domain(
             location,
         ))),
-        Err(error) => Ok(UpdateLocationResponse::Error(error.into())),
```
Comment (clemens-msupply) [inline graphql/location/src/mutations/update.rs:45]: You probably can remove our From implementation for the error... (the one that .into() uses)

**Review by clemens-msupply — APPROVED** [review]
Looks good thanks! just have a look if you can remove the error From implementations they should be unused now...

File: service/src/location/mod.rs (hunk 1 above)
Comment (roxy-dao) [reply service/src/location/mod.rs:1]: Need it to import the Enums... I can move just the enums out to public

Comment (roxy-dao) [reply service/src/location/mod.rs:1]: Will change back~

**Comment by clemens-msupply** [comment]
Some tests for the errors are failing now, should be straight forward to update (have a look for other tests which test std err)...

**Pull request merged**